api/bin/chainflip-ingress-egress-tracker/src/main.rs

```diff
@@ -22,24 +21,13 @@ async fn start(
 		.expect("setting default subscriber failed");
 
 	let client = redis::Client::open(settings.redis_url.clone()).unwrap();
-	let mut store = RedisStore::new(client.get_multiplexed_tokio_connection().await?);
+	let store = RedisStore::new(client.get_multiplexed_tokio_connection().await?);
 
-	// Broadcast channel will drop old messages when the buffer is full to
-	// avoid "memory leaks" due to slow receivers.
-	const EVENT_BUFFER_SIZE: usize = 1024;
-	let (witness_sender, _) =
-		tokio::sync::broadcast::channel::<state_chain_runtime::RuntimeCall>(EVENT_BUFFER_SIZE);
```

Comment:
no longer need to multicast the witnessing calls to different websockets so i removed the channel logic